universal-login-contracts/test/contracts/ERC1077-gasCost.js

```diff
@@ -9,10 +9,10 @@ import {getExecutionArgs} from '../utils';
 chai.use(chaiAsPromised);
 chai.use(solidity);
 
-const callCost = 109500;
-const transferCost = 125000;
+const callCost = 129500;
+const transferCost = 225000;
```

Comment:
This is part of continues performance practice - a test that makes sure we don't increase the cost of (already pretty expensive)  transaction significantly. 
We should:
- consider optimisations to keep single signature cost at the same/similar level
- add a test for multisignature case